File: Value equal to index value/Value_equal_to_index_value.py
class Solution:
    # Function to find values in array equal to their indices
    def valueEqualToIndex(self, arr):
        ans = []
        for i in range(1, len(arr) + 1):
            if arr[i - 1] == i:
                ans.append(arr[i - 1])
        return ans

File: Value equal to index value/Value_equal_to_index_value.cpp
class Solution {
  public:
    
    vector<int> valueEqualToIndex(vector<int>& arr) {
        vector<int> ans;
        int n = arr.size();
        for(int i =1; i<=n; i++) {
            if(arr[i-1] == i) ans.push_back(arr[i-1]);
        }
        return ans;
    }
};